meta-mender-imx/README.md

```diff
@@ -0,0 +1,58 @@
+# meta-mender-imx
+
+Mender integration for imx based boards
+
+The supported and tested boards are:
+
+ - [i.MX8M Nano EVK](TBD)
+
+Visit the individual board links above for more information on status of the
+integration and more detailed instructions on how to build and use images
+together with Mender for the mentioned boards.
+
+## Dependencies
+
+This layer depends on:
+
+```
+URI: https://github.com/Freescale/meta-freescale-3rdparty
+branch: zeus
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale-distro
+branch: zeus
+revision: HEAD
+```
+
+```
+URI: https://github.com/Freescale/meta-freescale
+branch: zeus
+revision: HEAD
+```
+
+## Quick start
+
+The following commands will setup the environment and allow you to build images
+that have Mender integrated.
+
+
+```
+mkdir mender-imx && cd mender-imx
+repo init -u https://github.com/mendersoftware/meta-mender-community \
+          -m meta-mender-imx/scripts/imx-5.4.3-2.0.0_demo_mender.xml \
+          -b zeus
+repo sync
+source setup-environment imx
+MACHINE=imx8mnevk bitbake core-image-base
```

Comment:
I think there is a missing step here, I did something like this for the board I am working on,

```
mkdir mender-tn-imx-bsp && cd mender-tn-imx-bsp
repo init -u https://github.com/TechNexion/tn-imx-yocto-manifest.git -b zeus_5.4.y-next -m imx-5.4.24-2.1.0.xml
wget --directory-prefix .repo/manifests https://raw.githubusercontent.com/mendersoftware/meta-mender-community/zeus/meta-mender-tn-imx-bsp/scripts/imx-5.4.24-2.1.0_demo_mender.xml
repo init -m imx-5.4.24-2.1.0_demo_mender.xml
repo sync -j$(nproc)
DISTRO=fsl-imx-xwayland MACHINE=pico-imx8mm source tn-setup-mender.sh -b build-xwayland-imx8mm
bitbake core-image-base
```

Reply:
I am assuming that we will get the mender specifics into the imx-manifest repo.  If that doesn't happen then we will need the local_manifest stuff.